src/shared/containers/GetOrganizations.tsx

```diff
@@ -89,10 +107,10 @@ const GetOrganizations: FunctionComponent = () => {
               />
               <Route path="/orgs" component={App} />
               <Route exact path="/" component={RouteToOrg} />
-              {CLOUD && canAccessCheckout(me) && (
+              {CLOUD && canAccessCheckout(quartzMe) && (
```

Comment:
what's the difference between using `quartzMe` and `me` here?

Reply:
It has always been an object related to `quartzMe` but was previously named `me`. Yes, very confusing.